### DISCUSSION TITLE
Ability to group/categorise Automations (maybe folders)

### DISCUSSION THREAD

**Discussion by rohangopal2** [home-assistant/frontend#16211] [Automations / scripts / scenes]
I cannot find a way to structure my automations. As you would imagine, the list of automations can get to hundreds - and it just seems to be listed alphabetically etc. - would love to be able to group it as needed etc. (for easy access and visibility and access) - THANKS

**Comment by frenck** [comment]
This is a duplicate of many others. Please use the search before creating something.

This one, particularly, is tracked here: <https://community.home-assistant.io/t/grouping-automations-on-frontend-for-organization/102771>
